### PR TITLE
Fixing memory and goroutine leak when connecting to standby DB

### PR DESCRIPTION
**Pull request by djpasch**

### CODE REVIEW

File: oci8.pc [hunk 1]
```diff
@@ -1,7 +1,7 @@
-libdir=/usr/lib/oracle/18.3/client64/lib
-includedir=/usr/include/oracle/18.3/client64
+libdir=/usr/lib/oracle/12.2/client64/lib
```
[the remark this answers is not in the document]
Comment (patrickschilderING) [reply oci8.pc]: I don't have that version running on my environments, so i won't be able to test it properly.
Can you accept the code change and then test it with the updated client library?

[the remark this answers is not in the document]
Comment (Yannig) [reply oci8.pc]: I wondering if I may release two versions of the same exporter. One compiled against 12.2 and the other one with last versions.

[the remark this answers is not in the document]
Comment (patrickschilderING) [reply oci8.pc]: I think it's good to have two releases of the same exporter, hereby we increase the compatibility.

[the remark this answers is not in the document]
Comment (Yannig) [reply oci8.pc]: It's good for me. I'll work on this idea in the next days.

Thanks!

**Pull request merged**